src/client/app/shared/canvas/canvas.component.css

```diff
@@ -0,0 +1,5 @@
+svg#svg {
+  width: 800px;
+  height: 800px;
```

Comment:
The problem is that there's the size of the SVG element and then there's the size of the coordinate space within it. To deal with this, I need to figure read up on how the viewBox property on the SVG element works. Hopefully it won't take too long, but I just kept it as a fixed size for now for simplicity's sake.